banyand/stream/elements.go

```diff
@@ -34,13 +34,15 @@ type tagValue struct {
 	tag       string
 	value     []byte
 	valueArr  [][]byte
+	primitive int64
```

Comment:
```suggestion
```

You can use `bytes.Compare` to compare `value`. `valueArr` doesn't support the skip index.